### ISSUE TITLE
Don't use “Unicode small-caps”

### ISSUE THREAD

**Issue by lynn** [stefichjo/toki-pona#107]
toki!

Use of “Unicode small-caps characters” like ᴀᴅᴊᴇᴄᴛɪᴠᴇ in this example is not a very good idea:

> The consonant starting off an ᴀᴅᴊᴇᴄᴛɪᴠᴇ is voiced.

It causes several problems:

* On my Mac, VoiceOver reads this as “The consonant starting off an is voiced.”
* If I search the page (Ctrl+F) for `adjective`, ᴀᴅᴊᴇᴄᴛɪᴠᴇ is not matched.
* Many fonts do not support the glyphs in “ᴀᴅᴊᴇᴄᴛɪᴠᴇ” and will render this as something like:
  > “The consonant starting off a ⍰⍰⍰⍰⍰⍰⍰⍰⍰ is voiced.”

Basically, these characters are not meant to be used for writing English words. They are part of the “Phonetic Extensions” Unicode block, and computers simply don't handle those the same as plain old lowercase letters. See [this question on SuperUser](https://superuser.com/q/1160295/1007879) for more information.

You should use markup on the word “adjective” instead:

> The consonant starting off an **adjective** is voiced.
>
> The consonant starting off an _adjective_ is voiced.
